miracl/stats/miracl_stats_ace_clusterwise.py

```diff
@@ -339,30 +344,34 @@ def main(args, output_dir_arg):
         atl_dir = Path("/code/atlases/ara/template")
         ann_dir = Path("/code/atlases/ara/annotation")
 
-    mask_filename = f"average_template_{img_res}um_brainmask.nii.gz"
-    ann_filename = f"annotation_hemi_combined_{img_res}um.nii.gz"
+    if hemi == "combined":
+        mask_filename = f"average_template_{img_res}um_brainmask.nii.gz"
```

Comment:
are the hemi brain templates for 10 and 25 um added to atlas repo of the miracl?

Reply:
It is not there for 10um you are right. I'm not sure how we add that in ther though.

Reply:
I can integrate the 10um templates into the Docker build time and add them to our Dropbox repo. @anthonyprinaldi where are they stored currently?

Reply:
Thanks @jono3030. I will check one of the containers, but I'm not sure we've ever used it or even created it, so I may have to create it now.

Reply:
I guess we don't need to add them. We are writing the files, not reading them, so the files don't need to exist prior to running these scripts.

Reply:
So these are not ARA files that should be in the annotation folder by default?

Reply:
There are a few that are there by default. Those already in the dropbox are good. But the ones that this comment addresses are for writing, not reading. 

The ones that we should add are the two that I shared over slack (for now until we get the other resolutions working too).

Reply:
Got it! The once you shared on Slack will be added to the Dropbox. I'll test the compilation once they were added and update you on Slack.